docs/sources/panels/visualizations/graph-panel.md

```diff
@@ -15,7 +15,7 @@ This visualization is the most-used in the Grafana ecosystem. It can render as a
 ## Data and field options
 
 Graph visualizations allow you to apply:
-- [Data transformations]({{< relref "../transformations.md" >}})
+- [Data transformations]({{< relref "../transformations/prereq.md" >}})
```

Comment:
I don't think linking to the prereq.md topic is a good idea. As-is, it is just a list of topics. I would link these to the Transformations overview topic.

Reply:
Moved to Overview as suggested.